test/commands/auth/jwt/grant.nut.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as path from 'path';
+import { execCmd, TestSession, prepareForJwt } from '@salesforce/cli-plugins-testkit';
+import { expect } from 'chai';
+import { Env } from '@salesforce/kit';
+import { ensureString, getString } from '@salesforce/ts-types';
+import { AuthFields } from '@salesforce/core';
+import { Result, expectUrlToExist, expectOrgIdToExist, expectAccessTokenToExist } from '../../../testHelper';
+
+let testSession: TestSession;
+
+describe('auth:jwt:grant NUTs', () => {
+  const env = new Env();
+  let jwtKey: string;
+  let username: string;
+  let instanceUrl: string;
+  let clientId: string;
+
+  before('prepare session and ensure environment variables', () => {
+    username = ensureString(env.getString('TESTKIT_HUB_USERNAME'));
+    instanceUrl = ensureString(env.getString('TESTKIT_HUB_INSTANCE'));
+    clientId = ensureString(env.getString('TESTKIT_JWT_CLIENT_ID'));
+    ensureString(env.getString('TESTKIT_JWT_KEY'));
+
+    testSession = TestSession.create({ authStrategy: 'NONE' });
+    jwtKey = prepareForJwt(testSession.homeDir);
+  });
+
+  after(async () => {
+    await testSession.clean();
```

Comment:
nit: I think `await testSession?.clean();` is a good pattern here because if something goes haywire with test session creation and it's not defined, this will fail too and it's just noise in the test results.